FluentTerminal.Models/Extensions.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentTerminal.Models.Enums;
+using Windows.ApplicationModel.Resources;
+
+namespace FluentTerminal.Models
+{
+    public static class Extensions
+    {
+        public static string GetResource(this string resource) =>

```

Comment:
this is not really declarative in what it does

Reply:
It gets a string from the resources, based on the key.

Reply:
`I18N.Translate("foo")` was kind of clear, `"foo".GetResource()` imho not so much